packages/examples/create-react-app/package.json

```diff
@@ -2,9 +2,9 @@
   "name": "mm-sdk-cra",
   "version": "1.0.0",
   "private": true,
-  "homepage": "./",
+  "homepage": "https://metamask.github.io/metamask-sdk",
```

Comment:
the problem if you do that it wont work locally so it needs to be environmentalize

Reply:
It works locally but doesn't solve the CI part, I'm completely removing it until I find a proper solution.